internal/nix/nix.go

```diff
@@ -77,22 +76,15 @@ func (*Nix) PrintDevEnv(ctx context.Context, args *PrintDevEnvArgs) (*PrintDevEn
 		if insecure, insecureErr := IsExitErrorInsecurePackage(err, "" /*installable*/); insecure {
 			return nil, insecureErr
 		} else if err != nil {
-			safeArgs := make([]string, 0, len(cmd.Args))
-			for _, a := range cmd.Args {
-				if a == args.FlakeDir {
-					a = "<redacted path>"
-				}
-				safeArgs = append(safeArgs, a)
-			}
-			return nil, redact.Errorf("nix command: %s", redact.Safe(safeArgs))
```

Comment:
I fumbled this with the original PR. I was going to try and sanitize the individual args, but then decided not to and ended up with a weird in between implementation. The line was supposed to just be:

```go
redact.Errorf("nix command: %s", redact.Safe(cmd))
```

and not have the `safeArgs` stuff. 

Redacted error messages are not normally redacted and retain their normal message unless they're sent to Sentry. But here I modified the actual slice of args, so it ended up always redacted.

If this isn't blocking anything, I can just submit a fix PR later.

Reply:
I think I see. I need to learn about our redact logic more.

Assuming it's okay, I'm going to land this, just so that the Devbox release is unblocked. Since it's a small PR, perhaps you could re-submit with the better fix and if it works out timing-wise then it could be included in the release as well!